i6_models/assemblies/conformer/conformer_v2.py

```diff
@@ -0,0 +1,154 @@
+from __future__ import annotations
+
+__all__ = ["ConformerBlockV2Config", "ConformerEncoderV2Config", "ConformerBlockV2", "ConformerEncoderV2"]
+
+import torch
+from torch import nn
+
+from dataclasses import dataclass, field
+from typing import Tuple, List, Optional
+
+from i6_models.config import ModelConfiguration, ModuleFactoryV1
+from i6_models.assemblies.conformer import (
+    ConformerConvolutionV1,
+    ConformerConvolutionV1Config,
+    ConformerMHSAV1,
+    ConformerMHSAV1Config,
+    ConformerPositionwiseFeedForwardV1,
+    ConformerPositionwiseFeedForwardV1Config,
+)
+
+
+@dataclass
+class ConformerBlockV2Config(ModelConfiguration):
+    """
+    Attributes:
+        ff_cfg: Configuration for ConformerPositionwiseFeedForwardV1
+        mhsa_cfg: Configuration for ConformerMHSAV1
+        conv_cfg: Configuration for ConformerConvolutionV1
+        modules: List of modules to use for ConformerBlockV2,
+                       "ff" for feed forward module, "mhsa" for multi-head self attention module, "conv" for conv module
```

Comment:
Indentation looks off. I think it should be indented by 4 spaces.
```suggestion
        modules: List of modules to use for ConformerBlockV2,
            "ff" for feed forward module, "mhsa" for multi-head self attention module, "conv" for conv module
```